packages/sampleOS/02_upgrades.yaml

```diff
@@ -26,25 +26,25 @@ stages:
                      debug: false
                      spinner_charset: 9
                   repositories:
-                  - name: "cos"
-                    description: "cOS official"
+                  - name: "sampleos"
+                    description: "sampleOS"
                     type: "docker"
                     enable: true
                     cached: true
                     priority: 1
                     verify: true
                     urls:
-                    - "raccos/releases-:FLAVOR:"
+                    - "raccos/releases-sampleos"
          permissions: 0600
          owner: 0
          group: 0
      - name: "Setup upgrade channel"
        files:
-       - path: /etc/cos-upgrade-image
+       - path: /etc/sampleos-upgrade-image
          permissions: 0600
          owner: 0
          group: 0
-       environment_file: /etc/cos-upgrade-image
```

Comment:
This will likely break upgrades, currently it's hardcoded here: https://github.com/rancher-sandbox/cOS-toolkit/blob/master/packages/installer/upgrade.sh#L62